json_rpc/clients/httpclient.nim

```diff
@@ -142,7 +142,7 @@ method call*(client: RpcHttpClient, name: string,
     id = client.getNextId()
     reqBody = requestTxEncode(name, params, id)
 
-  debug "Sending message to RPC server",
+  trace "Sending message to RPC server",
```

Comment:
broadly, this logging is currently needed for nimbus - what could be done is to implement a hook that would be called before/after each request - in such a world, we could remove logging entirely from json-rpc (notably, logging is used when the API does not expose enough information for the consumer of the api to log it by themselves - it's a shortcut to avoid complicating the API)